src/org.xtuml.bp.model.compare.test/src/org/xtuml/bp/model/compare/test/ElementOrderingTests.java

```diff
@@ -1125,7 +1142,24 @@ private void performMergeTest(IFile file) throws CoreException {
 			IFileState state = history[0];
 			InputStream contents = state.getContents();
 			copy = file.getProject().getFile(file.getName());
-			copy.create(contents, true, new NullProgressMonitor());
+			if (copy.exists()) {
+				try {
+					copy.delete(true, new NullProgressMonitor());
+				} catch (Throwable t) {
+					while (PlatformUI.getWorkbench().getDisplay().readAndDispatch()) ;
+				}
+				while (copy.exists()) {
+					while (PlatformUI.getWorkbench().getDisplay().readAndDispatch()) ;
+				}
+			}
+			try {
+				copy.create(contents, true, new NullProgressMonitor());				
+			} catch (Throwable t) {
+				boolean debug = true;

```

Comment:
Need to remove this debug code, otherwise if this catch is hit on the server the test will hang.